examples/cube/src/main.rs

```diff
@@ -114,33 +114,35 @@ fn main() {
     let object = rend3::types::Object {
         mesh: mesh_handle,
         material: material_handle,
-        transform: glam::Mat4::IDENTITY,
+        transform: Mat4::IDENTITY,
     };
     // Creating an object will hold onto both the mesh and the material
     // even if they are deleted.
     //
     // We need to keep the object handle alive.
     let _object_handle = renderer.add_object(object);
 
+    let view_location = Vec3::new(3.0, 3.0, -5.0);
+    let view_direction: Vec3 = (Mat3A::from_euler(glam::EulerRot::YXZ, -0.55, 0.5, 0.0) * Vec3A::Z).into();
```

Comment:
We should use this matrix directly if we can instead of going through look_at. We should be able to just invert this matrix to get a usable rotation matrix. Feel free to bug me on the matrix if this isn't right.